charts/k8ez/values.yaml

```diff
@@ -54,11 +72,11 @@ podDisruptionBudget:
 service:
   type: ClusterIP
   port: 8080
+#  targetPort: 8080  # will use port by default
 
 # Configure resources it will be given with reasonable defaults
 resources:
   limits:
-    cpu: 1000m
```

Comment:
?

Reply:
I believe we established that we didn't want to set an upper cpu limit